external/storm-hbase/src/main/java/org/apache/storm/hbase/bolt/HBaseLookupBolt.java

```diff
@@ -40,51 +48,85 @@
  *
  */
 public class HBaseLookupBolt extends AbstractHBaseBolt {
```

Comment:
Please don't use tab characters for indentation.  We are still working on the exact style guides, but we try to keep the style that is currently in the file and it was using spaces before and I expect the style guide to include spaces instead of tabs.